lua/astronvim/utils/buffer.lua

```diff
@@ -148,10 +148,12 @@ function M.close(bufnr, force)
         return
       end
     end
-    require("mini.bufremove").delete(bufnr, force)
-  else
-    vim.cmd((force and "bd!" or "confirm bd") .. (bufnr == nil and "" or bufnr))
+    if #vim.fn.tabpagebuflist() > 1 then -- if only one buffer left in tab, run `:bdelete`
```

Comment:
```suggestion
    if not M.is_valid(bufnr) or #vim.t.bufs > 1 then
```

This seems like more the intended behavior we would want in this case